### DISCUSSION TITLE
Manage codespaces in the app

### DISCUSSION THREAD

**Discussion by HRTK92** [community/community#37697] [Mobile] [answered]
I need to be able to open codespace from Github mobile.

**Comment by jfullstackdev** [comment, marked as answer]
I'm afraid that's a big service to be added as feature request,

Codespace either run on the browser or VS Code Editor

the thing you can do is to simply log in to GitHub on the browser of your mobile so you can access your Codespace, although I'm not so sure if it can be run on mobile version of the browser
